docs/running_a_server/jars.md

```diff
@@ -33,11 +34,16 @@ keywords:
 Forge is a jar that adds a modding API for developers to tap into. It is much more expansive than Bukkit, as players also have to install mods on their end, allowing for more possibilites. It's recommended for mods and modpacks. Mostly used for older versions of the game. (pre1.12)
 
 
- 
+
 ### [Fabric](https://fabricmc.net/) ([Installation Guide](fabric-setup))
 Fabric is a jar that makes modding easier and smoother than Forge. Fabric is considered superior to Forge due to a better API that is more documented. It's mostly used for more modern versions of the game.   
 Fabric, with a few performance mods added, is recommended if you want to run a modern technical MC server.
 
+
+
+### [Neo-Forge](https://neoforged.net/) ([Installation Guide](neoforge-setup))
```

Comment:
For consistency, could we call it NeoForge? Same thing in a few other places. Looks great, otherwise, thanks!